### DISCUSSION TITLE
How to configure PackageDirectory

### DISCUSSION THREAD

**Discussion by TipluSen** [Eptagone/Vite.AspNetCore#49] [Q&A]
My `Program.cs` is located under `~/App` and my `package.json` under `~/Client`.

I've tried to setup a custom directory by setting `PackageDirectory` in the options but it seems not to work.

I've tried 
- `~/Client`
- `~/Client/package.json`
- `../Client`
- `../Client/package.json`

Thanks for the great work!

edit: typo

**Comment by Eptagone** [comment]
Hi, is your app like this?
- App
    - program.cs
- Client
    - package.json
   
Mmm, it's a little strange organization but maybe it could be possible.

**Reply by TipluSen** [reply]
Yes, its structured like this. Like i mentioned above, i've tried multiple variants but its not working.
